packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.tsx

```diff
@@ -50,47 +53,70 @@ interface ComboboxContentProps {
 }
 
 export function ComboboxContent(props: ComboboxContentProps): JSX.Element {
+  const optionsExist = props.options.length > 0;
   const {
     searchValue,
     setSearchValue,
+    selectedElement,
+    setSelectedElement,
     open,
     setOpen,
     popperRef,
     popperStyles,
     attributes,
     filteredOptions,
+    optionsListRef,
   } = useComboboxContent(props.options);
 
-  const optionsExist = props.options.length > 0;
+  useComboboxAccessibility(
+    open,
+    handleSelection,
+    popperRef,
+    filteredOptions,
+    optionsListRef,
+    selectedElement,
+  );
 
   const template = (
     <div
       ref={popperRef}
+      id="ATL-Combobox-Content"
       data-testid="ATL-Combobox-Content"
+      tabIndex={0}
       className={classnames(styles.content, !open && styles.hidden)}
       style={popperStyles.popper}
       {...attributes.popper}
     >
       <Search
```

Comment:
Should we perhaps add a label and visually hide it?

Reply:
so if you don't provide the `subjectNoun` prop it'll default to `Search` as the placeholder which does get read out by VoiceOver. on top of that since we're using `type="search"` on the input it also calls it out as a `search text field`

then, if you do provide a `subjectNoun` the placeholder becomes even more specific or contextual eg.

![image](https://github.com/GetJobber/atlantis/assets/11843215/fe2a3aac-5855-4453-9663-9c78c80a1945)

personally I feel like that's enough? to me the only thing missing is perhaps a description of what happens when you search, but that's going to get pretty verbose. if someone was already familiar with comboboxes, it's probably implied how the search would be used, but I have doubts about how many people think about comboboxes